apps/chat/src/components/Chat/Publish/ReviewApplicationDialogView.tsx

```diff
@@ -118,16 +121,31 @@ export function ReviewApplicationDialogView() {
             </span>
           </div>
         )}
-        <div className="flex gap-4">
-          <span className="w-[122px] text-secondary">
-            {t('Completion URL:')}
-          </span>
-          <span className="max-w-[414px] break-all text-primary">
-            {application?.function?.mapping?.[
-              FEATURES_ENDPOINTS.chat_completion
-            ] ?? application?.completionUrl}
-          </span>
-        </div>
+        {application?.completionUrl &&
+          isEmpty(application?.function?.mapping) && (
+            <div className="flex gap-4">
+              <span className="w-[122px] text-secondary">
+                {t('Completion URL:')}
+              </span>
+              <span className="max-w-[414px] break-all text-primary">
+                {application.completionUrl}
+              </span>
+            </div>
+          )}
+        {application?.function?.mapping &&
+          !isEmpty(application?.function?.mapping) && (
+            <ReviewApplicationPropsSection
+              label="Endpoints"
+              appProps={application.function.mapping}
+              propsNames={FEATURES_ENDPOINTS_NAMES}
+            />
+          )}
+        {application?.function?.env && !isEmpty(application?.function?.env) && (
```

Comment:
```suggestion
        {application?.function?.env && !isEmpty(application.function.env) && (
```

Reply:
Thank you for the suggestion!
Updated as suggested.